megatron/core/transformer/cuda_graphs.py

```diff
@@ -275,39 +310,16 @@ def format_mem_bytes(mem_bytes):
             elif g_idx % 100 == 0 or g_idx == len(cls.cudagraph_record) - 1:
                 logger.info(f"{g_idx}/{len(cls.cudagraph_record)}. {progress_str}")
```

Comment:
Not directly related to this PR, but we can only log these on the first / last rank to reduce log clutter?